src/Routes/MoviesList/components/PageSelector/styled.js

```diff
@@ -0,0 +1,34 @@
+import styled from "styled-components";
+
+export const PageButton = styled.button`
+    border: none;
+    margin: 6px;
+    padding: 8px;
+`
+
+export const PageButtonNext = styled.button`
+    border: none;
+    margin: 6px;
+    padding: 8px;
+    background-color: #D6E4FF;
+    &:hover {
+        background-color: #baccef;
+    }
+`
+
+export const PageCounter = styled.span`
```

Comment:
here, we need added : 
`
font-weight: 600;
font-size: 16px;
`  for " 1 " and " 500 "

Reply:
words " page " and " of "need some ` div` with:
`
 font-weight: 600;
color: #7E839A;
` (better with theme)

Reply:
ill add theese changes in the merge commit